src/app/loans/loans-view/loan-accounts-button-config.ts

```diff
@@ -133,6 +133,11 @@ export class LoansAccountButtonConfiguration {
             icon: 'coins',
             taskPermissionName: 'CREATE_GOODWILL_TRANSACTION',
           },
+          {
+            name: 'Payment Waiver',
+            icon: 'coins',
+            taskPermissionName: 'CREATE_GOODWILL_TRANSACTION',
```

Comment:
Please fix this permission for Payment Waiver

Reply:
fixed.